### PR TITLE
Feature/vuetify 2.0

### LINKED ISSUE #56: Support Vuetify 2.0

### PR DESCRIPTION
**Pull request by ricardovanlaarhoven**
fixes #56

### CODE REVIEW

File: src/VuetifyResource.vue [hunk 1]
```diff
@@ -72,52 +72,63 @@
...
+                        <template v-slot:activator="{on}">
+                            <v-btn
+                                color="red"
+                                dark
+                                fab
+                                slot="activator"
+                                small
+                                v-if="canDeleteResources(selected) && selected.length >= 1"
+                                v-on:click="showDeleteConfirmation()"
+                                v-on="on"
```
Comment (martn001) [inline src/VuetifyResource.vue]: Does this work with multiple v-on's?

File: src/VuetifyResource.vue [hunk 2]
```diff
@@ -58,10 +58,10 @@
             v-model="snackbar.active"
         >
             {{ snackbar.text }}
-            <v-btn @click.native="snackbar.active = false" dark flat>{{lang('close')}}</v-btn>
+            <v-btn @click.native="snackbar.active = false" dark text>{{lang('close')}}</v-btn>
         </v-snackbar>
 
-        <v-layout row wrap>
```
Comment (martn001) [inline src/VuetifyResource.vue]: Was this necessary to apply the new version 2.0?

Comment (ricardovanlaarhoven) [reply src/VuetifyResource.vue]: Yep, new vuetify version has another grid system. We've used row's the wrong way.

File: src/VuetifyResource.vue (hunk 1 above)
Comment (ricardovanlaarhoven) [reply src/VuetifyResource.vue]: It does work, and it's necessary for the `v-slot:activator`

**Pull request merged**